db/seeds/users.rb

```diff
@@ -1 +1,2 @@
-User.create(email: 'somedude@gmail.com', user_name: 'somedude', password: 'password', password_confirmation: 'password')
+User.create(email: 'somedude@gmail.com', user_name: 'somedude', password: 'password', password_confirmation: 'password', admin: false)
```

Comment:
Line is too long. [134/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.